src/SFA.DAS.AdminService.Web/ViewModels/CertificateDelete/CertificateSubmitDeleteViewModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Newtonsoft.Json;
+using SFA.DAS.AssessorService.Domain.Consts;
+using SFA.DAS.AssessorService.Domain.Entities;
+using SFA.DAS.AssessorService.Domain.JsonData;
+
+namespace SFA.DAS.AdminService.Web.ViewModels.CertificateDelete
+{
+    public class CertificateSubmitDeleteViewModel : CertificateBaseViewModel, ICertificateViewModel
+    {
+        public long Uln { get; set; }
+        public string CertificateReference { get; set; }
+        public int StandardCode { get; set; }
+        public bool? IsDeleteConfirmed { get; set; }
+        public string SearchString { get; set; }
+        public int Page { get; set; }
+
+        public void FromCertificate(Certificate cert)
+        {
+            BaseFromCertificate(cert);
```

Comment:
Same as above comment.

Reply:
These are inheriting from ICertificateViewModel. Hence i cannot move them to the base class.